src/test/java/com/microsoft/azure/functions/worker/broker/JavaMethodExecutorTest.java

```diff
@@ -0,0 +1,61 @@
+package com.microsoft.azure.functions.worker.broker;
+
+import com.microsoft.azure.functions.worker.WorkerLogManager;
+import org.junit.*;
+
+import static junit.framework.TestCase.*;
+
+import java.io.File;
+import java.util.*;
+
+
+import com.microsoft.azure.functions.rpc.messages.*;
+import com.microsoft.azure.functions.worker.description.*;
+import com.microsoft.azure.functions.worker.reflect.*;
+
+public class JavaMethodExecutorTest {
+
+	private static FunctionClassLoaderProvider functionClassLoaderProvider = new FunctionClassLoaderProvider();
+
+	//TODO: figure out why below logics not always works on azure devops pipelines, it seems more to be a issue from
+	//	azure devops pipelines. Currently, all test cases are testing the TestFunctionsClass using system classloader as
+	// TestFunctionsClass is in the classpath when running the unit tests.
+
+//	@BeforeClass
```

Comment:
Let us discuss about this test case and format this class.

Reply:
This test case issue was resolved by adding logics to reset the singleton object before test case.